src/Traits/Trackable.php

```diff
@@ -27,9 +27,9 @@ public static function bootTrackable(): void
         });
     }
 
-    public function trackableUserResolver(): ?int
+    public function trackableOwnerResolver(): ?Model
```

Comment:
I think we should replace this function with `abstract public function trackableOwnerResolver(): ?Model;`. That way this can be validated during static analysis rather than during runtime.

Reply:
Ah I didn't know you could make functions in a trait abstract. Updated 👍🏽